README.adoc

```diff
@@ -122,13 +122,13 @@ See the Hibernate documentation for details.
 
 You must begin with either `create` or `update`, because you do not yet have the database
 structure. After the first run, you can switch it to `update` or `none`, according to
-program requirements. Use `update` when you want to make some change to the database
+program requirements. Use `update` when you want to make some changes to the database
 structure.
 
 The default for `H2` and other embedded databases is `create-drop`. For other databases,
 such as `MySQL`, the default is `none`.
 
```

Comment:
If this sentence needs another article, I would say it is before “production state”, not here.

Reply:
Ok, gonna change that.